com.sap.cloud.lm.sl.cf.shutdown.client/src/main/java/com/sap/cloud/lm/sl/cf/shutdown/client/configuration/EnvironmentBasedShutdownConfiguration.java

```diff
@@ -14,6 +14,8 @@
     private static final String CFG_USERNAME = "SHUTDOWN_USERNAME";
     private static final String CFG_PASSWORD = "SHUTDOWN_PASSWORD";
 
+    private static final String URL_IS_NOT_SPECIFIED_IN_APP_ENV = "{0} is not specified in the application's environment.";
```

Comment:
This constant is not always used for URLs.

Reply:
Changed to ARG_0_...